pkg/permissionclaims/utils.go

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
move to pkg/apis/v1alpha1/permissionclaims

Reply:
resolved